cukes-rest/src/main/java/lv/ctco/cukesrest/internal/context/BaseContextHandler.java

```diff
@@ -9,10 +9,14 @@
 
     protected List<String> extractGroups(String pattern) {
         List<String> allMatches = new ArrayList<String>();
-        Matcher m = Pattern.compile(GROUP_PATTERN).matcher(pattern);
-        while (m.find()) {
-            allMatches.add(m.group(1));
+
+        if (pattern != null) {
```

Comment:
TBH, after looking into it further, I think maybe the better approch would be to create 2 seperate steps. One that includes an operator and another that doesn't.

Example:
`^response contains an array "(.+)" of size (\d+)`
`^response contains an array "(.+)" of size (>=|>|<=|<) (\d+)`

This would be cleaner and also generate a proper/valid JSON report.

Right now, without seperate steps the generated JSON report is either:
```json
{
  "arguments": [
    {
      "val": "gadgets",
      "offset": 28
    },
    {
      "val": "1",
      "offset": 45
    }
  ],
  "location": "ThenSteps.response_Body_Contains_Array_With_Size(String,String,Integer)"
}
```

or 
```json
{
  "arguments": [
    {
      "val": "gadgets",
      "offset": 28
    },
    {},
    {
      "val": "1",
      "offset": 45
    }
  ],
  "location": "ThenSteps.response_Body_Contains_Array_With_Size(String,String,Integer)"
}
```

This is because `Match#toMap()` removes arguments whose values are empty.